feature/player/src/main/java/dev/anilbeesetti/nextplayer/feature/player/utils/PlayerGestureHelper.kt

```diff
@@ -251,6 +254,26 @@ class PlayerGestureHelper(
         }
     )
 
+    private val longPressHoldGestureDetector = GestureDetector(
```

Comment:
Maybe merge this into `longPressHoldGestureDetector`

Reply:
i tried it, but didn't work..